onnx/defs/nn/defs.cc

```diff
@@ -235,6 +235,7 @@ The convolution transpose operator consumes an input tensor and {filter_desc},
 and computes the output.)DOC";
             ReplaceAll(doc, "{filter_desc}", filter_desc);
             schema.SetDoc(doc);
+            schema.SinceVersion(2);
```

Comment:
for ONNX domain ops, the current max version is "2", so you should bump the version to “3” here. Check schema.h map_[ONNX_DOMAIN] = std::make_pair(1, 2); please.

Reply:
After thinking about it, I feel this change should not break anything... hmm, probably I should just add this change without bumping up the version number? If so, I should not add anything to old.cc.

Reply:
For the existing models, this change is fine. But for the new models with output_padding/adjs attribute, this may break the backend. If we think in this way, we can consider it as a breaking change.

Reply:
@houseroad There will always be operators/attributes that backends don't support. A breaking change is when something that was *previously* supported now has a different meaning (true breaking change.)

I'll check and see if we've explained this adequately in VERSIONING

Reply:
I reread VERSIONING and it looks like it supports your interpretation, so it looks like something got missed in translation.

Reply:
Hmmm, frankly speaking, I'm hesitating on this. 
With bumping version in this case, it's reasonable in terms of semantics, but it's kind of annoying as the version will be bumped frequently. 
Without bumping it, the case Lu mentioned will happen. A backend supports ONNX OP version=2 will NOT be able to load some models which use this new attribute. This looks bad, I think.

Reply:
@linkerzhang Remember that the backend will always not be able to understand the new attribute, no matter what you do. So it doesn't seem like much of a benefit to bump the version, except maybe an earlier test that there's a problem.